Doc/howto/clinic.rst

```diff
@@ -626,8 +623,8 @@ array) would be ``file``, but the C variable would be named ``file_obj``.
 You can use this to rename the ``self`` parameter too!
 
 
-Converting functions using PyArg_UnpackTuple
---------------------------------------------
+How to convert functions using PyArg_UnpackTuple
+------------------------------------------------
```

Comment:
```suggestion
How to convert functions using ``PyArg_UnpackTuple``
----------------------------------------------------
```

Should this have some markup?

Reply:
Yeah, that makes sense (also for the other cases you pointed out). We _could_ use C function markup here, but I'm not sure it matters that much in a heading.